index.html

```diff
@@ -180,5 +180,44 @@ <h1>About me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="contact-section">
+           <p class="contact-par">I'm always interested in hearing about new projects, so if you'd like to chat get in touch</p>
+            <form>
+                <ul class="first-item-inner-about">
+                    <li>
+                        <li>
+                            <label for="fullname"></label>
+                            <input id="fullname" type="text" name="full_name" placeholder="fullname" size="30" required>
+                        </li>
+                        <li>
+                            <label for="email"></label>
+                            <input id="email" type="email" name="email_name" placeholder="email" size="25" required>
+                        </li>
+                        <li>
+                            <label for="textarea"></label>
+                            <textarea id="textarea" type="textarea" name="textarea" size="500" required>Enter your comment here...</textarea>
+                
+                        </li>
+                       
+                    </li>
+                </ul>
+            <button class="button"> <a type="submit">Get in touch</a></button>
+            </form>
+            <div>
+                <social>
+                    <ul class="social-icons">
+                        <li><img src="media/github.svg"></li>
+                        <li><img src="media/twitter.svg"></li>
+                        <li><img src="media/Linkedin icon.svg"></li>
+                        <li><img src="media/medium.svg"></li>
+                        <li><img src="media/angellist.svg"></li>
+                    </ul>  
+                </social>
+            </div>
+            <a  class= "email" href="#" id="emailing">kdanalex69@gmail.com</a>
```

Comment:
Kindly make sure your social icons for the contact form are visible in the mobile version and that there are styles like those shown in the Figma design. Follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) to the Figma design :grinning:

Reply:
Done.